### DISCUSSION TITLE
How to setup public peer name?

### DISCUSSION THREAD

**Discussion by ziftech** [Ride-The-Lightning/RTL#999] [Q&A] [answered]
Hi,

Could you please suggest, how to setup public peer name for the channels?

![image](https://user-images.githubusercontent.com/7998566/163987853-c7e1bf23-c964-46f1-8ea7-1af959b3d64a.png)

**Comment by saubyk** [comment, marked as answer]
Hi @ziftech 
This is the node alias name which you can configure via your lightning config file.
If you're running LND, it should be the `LND.conf` file
If you're running Core Lightning, it should be the `config` file
If you're running Eclair, it should be the `eclair.conf` file

Hope that helps.
